styles.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+
+*,
+::before,
+::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0.2);
+  font-family: inter, Arial, Helvetica, sans-serif;
+}
+
+main {
+  width: 30%;
```

Comment:
- _(optional)_ You really did a good job with the UI but I recommend not to give a 30% to the main component because by doing that it gives a mobile view on desktop but on mobile, the view is not very good. You can also proceed by using a `media query`
- this is how it looks on mobile
![image](https://user-images.githubusercontent.com/37901469/164702359-bd53e710-5ad8-4d8e-aea4-c4debe833b43.png)